### ISSUE TITLE
Error attaching ISO: VBoxManage error: VBoxManage: error: Could not get the storage format of the medium

### ISSUE THREAD

**Issue by shibumi** [hashicorp/packer#5245]
packer version: 1.0.3
Host Platform: Arch Linux
template.json: https://github.com/archlinux/arch-boxes/blob/master/vagrant.json
output: https://paste.archlinux.de/M2D8/
virtualbox version: 5.1.26-1

**Comment by mwhooker** [comment]
Sorry, I'm not able to reproduce this issue with your json on my platform (osx). It would be helpful if you could post the full debug log with `PACKER_LOG=1` set.

```
2017/08/23 19:10:11 ui: ==> virtualbox-iso: Creating hard drive...
==> virtualbox-iso: Creating hard drive...
2017/08/23 19:10:11 packer: 2017/08/23 19:10:11 Executing VBoxManage: []string{"createhd", "--filename", "output-virtualbox-iso/packer-virtualbox-iso-1503540284.vdi", "--size", "20480", "--format", "VDI", "--variant", "Standard"}
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stdout: Medium created. UUID: ad3d8f1c-9706-466f-8ae7-262a9c136584
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stderr: 0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Executing VBoxManage: []string{"storagectl", "packer-virtualbox-iso-1503540284", "--name", "IDE Controller", "--add", "ide"}
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stdout:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stderr:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Executing VBoxManage: []string{"storageattach", "packer-virtualbox-iso-1503540284", "--storagectl", "IDE Controller", "--port", "0", "--device", "0", "--type", "hdd", "--medium", "output-virtualbox-iso/packer-virtualbox-iso-1503540284.vdi", "--nonrotational", "off", "--discard", "off"}
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stdout:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stderr:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Executing VBoxManage: []string{"storageattach", "packer-virtualbox-iso-1503540284", "--storagectl", "IDE Controller", "--port", "0", "--device", "1", "--type", "dvddrive", "--medium", "/Users/mwhooker/dev/packertest/packer_cache/2bf448a9db7d9dd9cb08ad19e75f4acf6e5b257f2f67a7f8c4afb37e1e1c978b.iso"}
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stdout:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stderr:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Not attaching guest additions since we're uploading.
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Looking for available port between 5900 and 6000 on 127.0.0.1
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Trying port: 5913
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Executing VBoxManage: []string{"modifyvm", "packer-virtualbox-iso-1503540284", "--vrdeaddress", "127.0.0.1", "--vrdeauthtype", "null", "--vrde", "on", "--vrdeport", "5913"}
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stdout:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stderr:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 No floppy disk, not attaching.
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Looking for available communicator (SSH, WinRM, etc) port between 2222 and 4444
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Trying port: 2803
2017/08/23 19:10:12 ui: ==> virtualbox-iso: Creating forwarded port mapping for communicator (SSH, WinRM, etc) (host port 2803)
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Executing VBoxManage: []string{"modifyvm", "packer-virtualbox-iso-1503540284", "--natpf1", "packercomm,tcp,127.0.0.1,2803,,22"}
==> virtualbox-iso: Creating forwarded port mapping for communicator (SSH, WinRM, etc) (host port 2803)
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stdout:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stderr:
2017/08/23 19:10:12 ui: ==> virtualbox-iso: Executing custom VBoxManage commands...
==> virtualbox-iso: Executing custom VBoxManage commands...
2017/08/23 19:10:12 ui:     virtualbox-iso: Executing: modifyvm packer-virtualbox-iso-1503540284 --memory 1024
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Executing VBoxManage: []string{"modifyvm", "packer-virtualbox-iso-1503540284", "--memory", "1024"}
    virtualbox-iso: Executing: modifyvm packer-virtualbox-iso-1503540284 --memory 1024
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stdout:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stderr:
2017/08/23 19:10:12 ui:     virtualbox-iso: Executing: modifyvm packer-virtualbox-iso-1503540284 --cpus 2
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Executing VBoxManage: []string{"modifyvm", "packer-virtualbox-iso-1503540284", "--cpus", "2"}
    virtualbox-iso: Executing: modifyvm packer-virtualbox-iso-1503540284 --cpus 2
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stdout:
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stderr:
2017/08/23 19:10:12 ui: ==> virtualbox-iso: Starting the virtual machine...
==> virtualbox-iso: Starting the virtual machine...
2017/08/23 19:10:12 ui:     virtualbox-iso: The VM will be run headless, without a GUI. If you want to
    virtualbox-iso: view the screen of the VM, connect via VRDP without a password to
    virtualbox-iso: rdp://127.0.0.1:5913
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 Executing VBoxManage: []string{"startvm", "packer-virtualbox-iso-1503540284", "--type", "headless"}
    virtualbox-iso: The VM will be run headless, without a GUI. If you want to
    virtualbox-iso: view the screen of the VM, connect via VRDP without a password to
    virtualbox-iso: rdp://127.0.0.1:5913
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stdout: Waiting for VM "packer-virtualbox-iso-1503540284" to power on...
2017/08/23 19:10:12 packer: VM "packer-virtualbox-iso-1503540284" has been successfully started.
2017/08/23 19:10:12 packer: 2017/08/23 19:10:12 stderr:
2017/08/23 19:10:12 ui: ==> virtualbox-iso: Waiting 10s for boot...
==> virtualbox-iso: Waiting 10s for boot...
==> virtualbox-iso: Typing the boot command...
2017/08/23 19:10:22 ui: ==> virtualbox-iso: Typing the boot command...
```

**Comment by shibumi** [comment]
Thanks for the answer. The checksum for the iso has changed. That was the reason. See also: https://github.com/archlinux/arch-boxes/issues/19
